test/a2atests/TaskManagerTests.cs

```diff
@@ -39,7 +39,8 @@ public async Task CreateTask()
 
 
     [Fact]
-    public async Task CreateAndRetreiveTask() {
+    public async Task CreateAndRetrieveTask()
```

Comment:
You probably want to create an editorConfig file so that I don't keep changing these :-)